lib/index.js

```diff
@@ -76,6 +77,9 @@ class PWMetrics {
     if (this.opts.json) {
       return data;
     } else if (this.opts.expectations) {
+      if (this.opts.showChart) {
+        this.showChart(data);
+      }
       expectations.checkExpectations(data.timings, this.opts.metrics)
```

Comment:
I actually prefer the existing arrangement, and here's why. Whenever I'm running any script, the final thing I always want to see in the output are errors/warnings. Reason being is that it's easier to lose track of said errors/warnings if they're earlier on in such output, and much less likely lost if they are the last piece of output.

Also, it seems to me like the logical order of things is 1. see timings 2. see errors/warnings resulting from these timings.

With that said, my preference would be to keep as is. Thoughts?